src/components/Header.js

```diff
@@ -9,10 +9,10 @@ const Header = () => (
       <h1>Space Travelers&apos; Hub</h1>
     </div>
     <nav className="d-flex nav-links">
-      <NavLink to="/Rockets">Rockets</NavLink>
-      <NavLink to="/Missions">Missions</NavLink>
+      <NavLink to="/Rockets" className={({ isActive }) => (isActive ? 'styled-link' : 'no-styled-link')}>Rockets</NavLink>
+      <NavLink to="/Missions" className={({ isActive }) => (isActive ? 'styled-link' : 'no-styled-link')}>Missions</NavLink>
       <div className="line" />
-      <NavLink to="/MyProfile">My Profile</NavLink>
+      <NavLink to="/MyProfile" className={({ isActive }) => (isActive ? 'styled-link' : 'no-styled-link')}>My Profile</NavLink>
```

Comment:
You don't need to write this complicated lines here. In order to apply KISS rule, kindly style activated links just by adding active class to the CSS.